Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -837,6 +837,7 @@ public void RaiseSourceDetected(IMixedRealityInputSource source, IMixedRealityCo
             };
```

Comment:
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7550 updates the input system a bit.

@davidkline-ms and @vaoliva, it might be good to work together to make sure the new additions are instrumented well and tested for perf.

Reply:
I will work with @vaoliva to ensure that her changes merge correctly and have the markers.